### ISSUE TITLE
Null pointer exception

### ISSUE THREAD

**Issue by mabar** [mesour-intellij-plugins/latte-pro-issues#82]
**Describe the bug**

I just opened project ([orisai/source-map](https://github.com/orisai/source-map) in this case) and exception appeared. Project does not even contain Latte files.

**Environment:**
 - PhpStorm version [2022.3]
 - Plugin version [1.0.9]

**To Reproduce**

Not sure, it happens sometimes when I open project, not always

**Error**

```txt
Error executing task com.mesour.intellij.latte.configuration.c.f$$Lambda$3684/0x00000008019a0938@4ea6d847

java.lang.NullPointerException: Cannot invoke "com.mesour.intellij.latte.configuration.c.b$a.a()" because "<parameter1>.a" is null
	at com.mesour.intellij.latte.configuration.c.b.equals(SourceFile:92)
	at com.intellij.util.indexing.impl.ValueContainerImpl.getFileSetObject(ValueContainerImpl.java:331)
	at com.intellij.util.indexing.impl.ValueContainerImpl.addValue(ValueContainerImpl.java:72)
	at com.intellij.util.indexing.impl.ValueContainerImpl.associateValue(ValueContainerImpl.java:565)
	at com.intellij.util.indexing.impl.ValueContainerImpl.readFrom(ValueContainerImpl.java:515)
	at com.intellij.util.indexing.impl.ValueContainerExternalizer.read(ValueContainerExternalizer.java:30)
	at com.intellij.util.indexing.impl.ValueContainerExternalizer.read(ValueContainerExternalizer.java:12)
	at com.intellij.util.io.PersistentMapImpl.doGet(PersistentMapImpl.java:612)
	at com.intellij.util.io.PersistentMapImpl.get(PersistentMapImpl.java:549)
	at com.intellij.util.indexing.impl.ValueContainerMap.lambda$getModifiableValueContainer$0(ValueContainerMap.java:81)
	at com.intellij.util.indexing.impl.ChangeTrackingValueContainer.getMergedData(ChangeTrackingValueContainer.java:105)
	at com.intellij.util.indexing.impl.ChangeTrackingValueContainer.getValueIterator(ChangeTrackingValueContainer.java:85)
	at com.intellij.util.indexing.impl.MergedValueContainer$1.<init>(MergedValueContainer.java:27)
	at com.intellij.util.indexing.impl.MergedValueContainer.getValueIterator(MergedValueContainer.java:25)
	at com.intellij.util.indexing.impl.MergedValueContainer.getValueIterator(MergedValueContainer.java:11)
	at com.intellij.util.indexing.FileBasedIndexEx.lambda$processValueIterator$12(FileBasedIndexEx.java:440)
	at com.intellij.util.indexing.FileBasedIndexEx.lambda$processExceptions$7(FileBasedIndexEx.java:330)
	at com.intellij.util.ConcurrencyUtil.withLock(ConcurrencyUtil.java:233)
	at com.intellij.util.indexing.FileBasedIndexEx.lambda$processExceptions$8(FileBasedIndexEx.java:330)
	at com.intellij.util.indexing.diagnostic.IndexAccessValidator.validate(IndexAccessValidator.java:46)
	at com.intellij.util.indexing.FileBasedIndexEx.processExceptions(FileBasedIndexEx.java:328)
	at com.intellij.util.indexing.FileBasedIndexEx.processValueIterator(FileBasedIndexEx.java:443)
	at com.intellij.util.indexing.FileBasedIndexEx.processValuesInScope(FileBasedIndexEx.java:403)
	at com.intellij.util.indexing.FileBasedIndexImpl.processValuesInScope(FileBasedIndexImpl.java:1123)
	at com.intellij.util.indexing.FileBasedIndexEx.getValues(FileBasedIndexEx.java:139)
	at com.mesour.intellij.latte.configuration.c.a.b(SourceFile:76)
	at com.mesour.intellij.latte.configuration.c.a.a(SourceFile:56)
	at com.mesour.intellij.latte.configuration.c.a.a(SourceFile:171)
	at com.mesour.intellij.latte.configuration.c.f.a(SourceFile:39)
	at com.intellij.openapi.project.DumbServiceImpl.doRun(DumbServiceImpl.java:388)
	at com.intellij.openapi.project.DumbServiceImpl.updateFinished(DumbServiceImpl.java:380)
	at com.intellij.openapi.project.TrackedEdtActivityService$TrackedEdtActivity.run(TrackedEdtActivityService.java:66)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:813)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:429)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:74)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:114)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:36)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:909)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:756)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$5(IdeEventQueue.java:437)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:774)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:436)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:615)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:434)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:813)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:480)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

**Comment by mabar** [comment]
This still happens quite a lot when opening projects (v 1.0.10)

**Comment by mesour** [comment]
Fixed in 1.0.11
